presto-tests/src/test/java/com/facebook/presto/tests/TestTpchTaskUpdateSizeLimitIntegrationTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.facebook.presto.Session;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.tests.tpch.TpchQueryRunner.createQueryRunner;
+
+public class TestTpchTaskUpdateSizeLimitIntegrationTest
+        extends AbstractTestQueryFramework
+{
+    @SuppressWarnings("unused")
+    public TestTpchTaskUpdateSizeLimitIntegrationTest()
+    {
+        this(() -> createQueryRunner(ImmutableMap.of("experimental.internal-communication.max-task-update-size", "100B")));
+    }
+
+    protected TestTpchTaskUpdateSizeLimitIntegrationTest(QueryRunnerSupplier queryRunnerSupplier)
+    {
+        super(queryRunnerSupplier);
+    }
+
+    @Test
+    public void testTaskUpdateSizeLimit()
+    {
+        Session session = Session.builder(getSession()).build();
+        assertQueryFails(session, "SELECT * " +
+                        "FROM ORDERS " +
```

Comment:
Not need to have such complicated example. Just use `SELECT * FROM ORDERS LIMIT 10`